pandas/io/parsers.pyi

```diff
@@ -41,7 +41,7 @@ def read_csv(
     date_parser: Optional[Callable] = ...,
     dayfirst: bool = ...,
     cache_dates: bool = ...,
-    iterator: bool = ...,
+    iterator: Literal[True],
```

Comment:
This gives the error "Non-default argument follows default argument". I think this needs to be `iterator: Literal[False] = ...,`

Reply:
I think these are meant to be keyword-only parameters, so there's a missing "*" parameter to separate the positional from keyword-only params. If this is added, the error will go away.